Core/clim-basic/events.lisp

```diff
@@ -241,6 +241,27 @@
 (defmethod window-configuration-event-y ((event window-configuration-event))
   (get-window-position ((event-sheet event) event) y))
 
+(defgeneric window-configuration-event-width (window-configuration-event))
+
+(defmethod window-configuration-event-width ((event window-configuration-event))
+    (multiple-value-bind (width height)
+        (untransform-distance  (sheet-native-transformation
+                                (sheet-parent (event-sheet event)))
+                               (window-configuration-event-native-width event)
+                               (window-configuration-event-native-height event))
+      width))
+
+(defgeneric window-configuration-event-height (window-configuration-event))
+
+(defmethod window-configuration-event-height ((event window-configuration-event))
+  (multiple-value-bind (width height)
+        (untransform-distance  (sheet-native-transformation
+                                (sheet-parent (event-sheet event)))
+                               (window-configuration-event-native-width event)
+                               (window-configuration-event-native-height event))
+      height))
+
+
```

Comment:
please rewrite it to `(defgeneric … (:method …))` and stick to maximum one empty line between toplevel forms